<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>second</class>
 <widget class="QDialog" name="second">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>77</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelName1">
      <property name="text">
       <string>empty</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>goldan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="openButton1">
      <property name="text">
       <string>open</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="selectButton1">
      <property name="text">
       <string>select</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saleButton1">
      <property name="text">
       <string>sale</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>77</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="labelName2">
      <property name="text">
       <string>empty</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>goldan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="openButton2">
      <property name="text">
       <string>open</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="selectButton2">
      <property name="text">
       <string>select</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saleButton2">
      <property name="text">
       <string>sale</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>77</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="labelName3">
      <property name="text">
       <string>empty</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>goldan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="openButton3">
      <property name="text">
       <string>open</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="selectButton3">
      <property name="text">
       <string>select</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saleButton3">
      <property name="text">
       <string>sale</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>77</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="labelName4">
      <property name="text">
       <string>empty</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>goldan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="openButton4">
      <property name="text">
       <string>open</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="selectButton4">
      <property name="text">
       <string>select</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saleButton4">
      <property name="text">
       <string>sale</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>77</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="labelName5">
      <property name="text">
       <string>empty</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>goldan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="openButton5">
      <property name="text">
       <string>open</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="selectButton5">
      <property name="text">
       <string>select</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saleButton5">
      <property name="text">
       <string>sale</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
